### ISSUE TITLE
Cross-platform colors with go-colortext

### ISSUE THREAD

**Issue by frontierpsycho** [op/go-logging#110]
How about marrying this project with [go-colortext](https://github.com/daviddengcn/go-colortext) for easy, cross-platform colors?

**Comment by GwynethLlewelyn** [comment]
I agree totally; there are more alternatives, if the original developers do not like go-colortext for some reason, for instance [color](https://github.com/fatih/color), which is _very easy_ to retrofit to existing projects (and also supports the Windows console and not only ANSI escaping).

**Comment by GwynethLlewelyn** [comment]
Whoops. It seems that this is already implemented somehow: https://github.com/op/go-logging/pull/112

I guess this issue should be closed now...

**Comment by shenwei356** [comment]
but the author is too busy to merge it.
